website/docs/r/org.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org"
+sidebar_current: "docs-vcd-resource-org"
+description: |-
+Provides a vCloud Director Organization resource. This can be used to create and delete an organization.
+---
+
+# vcd\_org
+
+Provides a vCloud Director Org resource. This can be used to create and delete an organization.
+
+Supported in provider *v2.0+*
+
+## Example Usage
```

Comment:
I believe creation of an Org is only possible with the sys admin user (correct me, if you know of a way to do it with other user type). Hence, we need a sentence about this and it wouldn't hurt to add a `provider` sample in the example below with the correct user.